### ISSUE TITLE
thestar.co.uk: ads

### ISSUE THREAD

**Issue by rjdg14** [uBlockOrigin/uAssets#24459]
### Prerequisites

- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367), [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) or [_a shortener/hosting site_](https://github.com/uBlockOrigin/uAssets/discussions/17361) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I did not remove any of the [default filter lists](https://github.com/uBlockOrigin/uAssets/assets/66517106/6e8d024a-5869-47cf-b142-b2357eb6c7d0), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin along with other content blockers.
- [X] I have verified that the web browser's built-in content blocker/tracking protection, [network wide/DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking), or my VPN is not causing the issue.
- [X] I have verified that other extensions are not causing the issue.
- [X] If this is about a breakage or detection, I have verified that it is caused by uBlock Origin and isn't a site issue.
- [ ] I did not answer truthfully to **ALL** the above checkboxes.

### URL address of the web page

`https://www.thestar.co.uk/`

### Category

ads

### Description

The site appears to displaying ads despite uBlock Origin being enabled. If I manually attempt to block one of the ads, a new one simply reappears in the place of the previous one.

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![UBlockStarAds](https://github.com/uBlockOrigin/uAssets/assets/73315260/6d8848c7-bdd8-42dc-a354-c30268a7a00f)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.58.0
Chromium: 126
filterset (summary):
 network: 219274
 cosmetic: 256495
 scriptlet: 54351
 html: 0
listset (total-discarded, last-updated):
 added:
  https://filters.adtidy.org/extension/ublock/filters/14.txt: 80383-80383, 5m
  https://secure.fanboy.co.nz/fanboy-annoyance_ubo.txt: 100265-87370, 5m
  adguard-cookies: 30304-50, 6m
  adguard-generic: 83112-7370, 6m
  adguard-mobile: 9593-11, 6m
  adguard-mobile-app-banners: 5319-10, 5m
  adguard-other-annoyances: 14754-74, 5m
  adguard-popup-overlays: 27412-201, 5m
  adguard-social: 23103-1662, 6m
  adguard-spyware: 78703-30986, 6m
  adguard-spyware-url: 1585-6, 6m
  adguard-widgets: 2594-2, 5m
  [7 lists not shown]: [too many]
 default:
  user-filters: 870-54, never
  easylist: 86753-724, 6m
  easyprivacy: 52748-740, 6m
  plowe-0: 3550-83, 6m
  ublock-badware: 9171-2, 6m
  ublock-filters: 39192-666, 6m
  ublock-privacy: 1034-119, 6m
  ublock-quick-fixes: 138-2, 6m
  ublock-unbreak: 2386-5, 6m
  urlhaus-1: 16970-5, 6m
filterset (user): [array of 874 redacted]
trustedset:
 added: [array of 14 redacted]
switchRuleset:
 added: [array of 6 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 1079 ms (selfie)
 maxAssetCacheWait: 608 ms
 cacheBackend: indexedDB
popupPanel:
 blocked: 22
 network:
  amazon-adsystem.com: 1
  ccgateway.net: 1
  criteo.net: 1
  crwdcntrl.net: 1
  dotmetrics.net: 1
  edkt.io: 1
  googletagmanager.com: 1
  hs-scripts.com: 1
  html-load.com: 3
  npttech.com: 1
  onesignal.com: 1
  outbrain.com: 1
  parsely.com: 1
  privacymanager.io: 1
  rlcdn.com: 1
  rubiconproject.com: 1
  scorecardresearch.com: 1
  tinypass.com: 1
  userreport.com: 1
  viafoura.co: 1
 extended:
  ##.banner
  ##.billboard
  ##[data-type="AdRowBillboard"]
  ##div[class^="AdLoadingText"]
  ##div[data-ads-params]
  ##div[class*="AdWrapper"]
  ##div[class*="AdLoadingText"]
  ##.dmpu-ad
  ##+js(trusted-click-element, button[aria-label="I Accept"], , 10…
  ##+js(set-local-storage-item, adshield-analytics-uuid, $remove$)
  ##+js(set-local-storage-item, /_fa_dXVpZA==$/, $remove$)
  ##+js(set-local-storage-item, /_fa_Y2FjaGVfaXNfYmxvY2tpbmdfYWNjZ…
  ##+js(set-local-storage-item, /_fa_Y2FjaGVfaXNfYmxvY2tpbmdfYWRz$…
  ##+js(set-local-storage-item, /_fa_Y2FjaGVfYWRibG9ja19jaXJjdW12Z…
  ##+js(prevent-fetch, /static\.criteo\.net|adsbygoogle/, length:1…
```
</details>

**Comment by stephenhawk8054** [comment]
Test again in a new browser profile with only uBO and its default settings.

**Comment by JobcenterTycoon** [comment]
They started to bait some new requests.

Best would be to use [microShield](https://github.com/List-KR/microShield). its a userscript designed to block these ads.

**Comment by stephenhawk8054** [comment]
@JobcenterTycoon Do these work on your side?

```adb
||html-load.$important
*$doc,csp=script-src-attr 'none'
```

**Comment by JobcenterTycoon** [comment]
The filter need a few hours until they are avaiable as a update. But you can force uBlock Origin to fetch the latest filterlist by clicking here: https://ublockorigin.github.io/uAssets/update-lists.html?listkeys=ublock-filters&manual=1

"they started to bait some new requests" is a information for filterlist maintainer.

**Comment by rjdg14** [comment]
> `https://www.thestar.co.uk/`

I've just updated the filters via that list and unfortunately the ads are still there. Have the updates simply not been rolled out yet to the public filterlist?

**Comment by JobcenterTycoon** [comment]
@stephenhawk8054 for `thestar.co.uk` yes. But it will break for sites which using a Ad-Shield PHP part (like `loawa.com`) so every site need to be checked by hand.

**Comment by stephenhawk8054** [comment]
@JobcenterTycoon Yeah, I just mean for the sites using inline scripts approach, so we don't need to use exceptions for these sites.

**Comment by JobcenterTycoon** [comment]
Its hard to check the whole site for breakage. As i heard the filter are risky?

**Comment by rjdg14** [comment]
> @stephenhawk8054 for `thestar.co.uk` yes. But it will break for sites which using a Ad-Shield PHP part (like `loawa.com`) so every site need to be checked by hand.
> 
> @rjdg14 do you clicked on the link? It works fine for me now on Chrome and Firefox.

Yep. The ads are still there.

Something else I thought I should mention about the site's behaviour would be that as I mentioned earlier, if you block one of the ad banners manually with uBlock Origin, a new one will simply spawn in its place. After about 3 repeats of blocking these, the site suddenly reloaded without any CSS and I got this message:

![SheffieldStarError](https://github.com/user-attachments/assets/9e4330c5-7e17-4be3-9acf-f3e3a4e5e4c7)


I think that popup in the middle is from the website and neither Chrome nor uBlock Origin.




Not directly related to this site, but one fairly common anti-adblock tool used by a fair number of sites is made by a company called Admiral, and its technology seems to enforce itself via one of a thousand or so different domains mostly named weird combinations of two words. Blocking these enforcement domains seems to disable such popups.

**Comment by JobcenterTycoon** [comment]
@rjdg14 try

```
*$doc,csp=script-src-attr 'none',domain=thestar.co.uk|yorkshirepost.co.uk
||html-load.com/loader.min.js$domain=thestar.co.uk|yorkshirepost.co.uk
```

**Comment by rjdg14** [comment]
> *$doc,csp=script-src-attr 'none',domain=thestar.co.uk|yorkshirepost.co.uk
> ||html-load.com/loader.min.js$domain=thestar.co.uk|yorkshirepost.co.uk

Is this for my uBlock custom filters?

**Comment by JobcenterTycoon** [comment]
Yes. Add the filter to your custom filter and test if the filter works fine.

**Comment by rjdg14** [comment]
> Yes. Add the filter to your custom filter and test if the filter works fine.

It worked :) .

I suspect the owner of these sites has rolled out this change fairly recently, hence why its ads have been escaping the default uBlock Origin filters.

**Comment by JobcenterTycoon** [comment]
Added to the filterlist now.
